### PR TITLE
chore: remove screenshot options for firefox

### PR DESCRIPTION
**Pull request by jrandolf-google**

### CODE REVIEW

**Comment by OrKoN** [comment]
eslint is failing

File: packages/puppeteer-core/src/cdp/Page.ts [hunk 1]
```diff
@@ -1052,11 +1053,17 @@ export class CdpPage extends Page {
...
     } = options;
 
+    const isFirefox =
+      this.target()._targetManager() instanceof FirefoxTargetManager;
+
     await using stack = new AsyncDisposableStack();
-    if (omitBackground && (type === 'png' || type === 'webp')) {
+    // Firefox omits background by default; it's not configurable.
+    if (!isFirefox && omitBackground && (type === 'png' || type === 'webp')) {
       await this.#emulationManager.setTransparentBackgroundColor();
```
Comment (OrKoN) [inline packages/puppeteer-core/src/cdp/Page.ts:1062]: can we catch the error on the transparent color call instead of checking for Firefox?

**Review by OrKoN — APPROVED** [review]

File: packages/puppeteer-core/src/cdp/Page.ts (hunk 1 above)
Comment (jrandolf) [reply packages/puppeteer-core/src/cdp/Page.ts:1062]: I think your method is more "robust", but it's not like we will support other CDP clients.

Comment (jrandolf) [reply packages/puppeteer-core/src/cdp/Page.ts:1062]: In particular, the error will only exist in firefox, so why not just match by checking targetmanager?

**Pull request merged**